examples/importing/vapp-vm/terraform.tfvars.example

```diff
@@ -0,0 +1,7 @@
+vcd_url  = "https://some-vcd.com"
+org      = "myorg"
+vdc      = "myvdc"
+catalog  = "mycatalog"
```

Comment:
Maybe is it worth it to add the same comments as `examples/importing/catalog-contents/catalog-sharing/terraform.tfvars.example`? (To differentiate what must already exist from what should be created)

Reply:
Done